terasoluna-gfw-functionaltest-env/configs/tomcat10-oracle/resources/META-INF/spring/terasoluna-gfw-functionaltest-infra.properties

```diff
@@ -1 +1,2 @@
 database=ORACLE
+dialect=org.hibernate.dialect.Oracle12cDialect
```

Comment:
Oracle12cDialect is deprecated, so if you use it, it is OracleDialect(12), but this fix is a temporary fix, and we know we will remove it later, so we will leave it as is for now.
If later verification shows that it cannot be removed, please modify to use OracleDialect(12).

https://docs.jboss.org/hibernate/orm/6.1/javadocs/org/hibernate/dialect/package-summary.html

Reply:
Deleted at https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/pull/1041.